### DISCUSSION TITLE
Logarithmic axes

### DISCUSSION THREAD

**Discussion by raspberryananas** [microsoft/nni#3775] [Ideas]
Could you add an option where the user can decide whether the axes are recorded logarithmically? This would improve readability.

![image](https://user-images.githubusercontent.com/12211578/65488420-5ea19d80-dea9-11e9-8114-98bf68832e7c.png)

**Comment by xuehui1991** [comment]
@martsalz  Thank！It's really a good suggestion!

**Comment by scarlett2018** [comment]
@kvartet - may you help on making an initial design with junzhu for team review? thanks.

**Comment by kvartet** [comment]
hello @martsalz, sorry for my late reply and thanks for your suggestion again. I want to confirm with you, do you want to decide whether the metric axes are recorded logarithmically in the Intermediate Result Graph? 
![image](https://user-images.githubusercontent.com/48014605/121490514-aebaed80-ca07-11eb-9576-6b65efda16bb.png)
